config/runtime.exs

```diff
@@ -0,0 +1,106 @@
+import Config
+
+if config_env() != :test do
```

Comment:
ideally we should test the env var loading by having an `.test.env` checked into the repo. that would ensure that the env var passing to config is correct.

Reply:
we can do this in another PR, not as important.

Reply:
No need, it works without it and this works for now, we can refine as we go for now